cli/cook/subcommands/kill.py

```diff
@@ -0,0 +1,109 @@
+from cook import http, colors
+from cook.querying import query, print_no_data
+from cook.util import strip_all, print_info
+
+
+def guard_against_duplicates(uuids, query_result):
+    """
+    Checks for UUIDs that are duplicated and throws if any are found (unlikely to happen in the wild, but it could)
+    """
+    if query_result['count'] == 1:
+        return
+
+    uuid_to_entries = {}
```

Comment:
You can use a `defaultdict` here to avoid the if/else logic in `add`

Reply:
Nice, done